examples/cemetery.ttl

```diff
@@ -90,6 +90,57 @@
            schema:ratingValue 2 ;
            schema:worstRating 1 .
 
+# Enrichment
+<Mention/1> a wev:Mention ;
+                     schema:description "Calais" ;
+                     wev:confidence "0.93"^^xsd:decimal ;
+                     wev:startOffset "29"^^xsd:integer ;
+                     wev:endOffset "34"^^xsd:integer ;
+                     schema:subjectOf <Appearance/1> ;
+                     wev:hasConcept <Location/1> ;
+                     .
+
+<Location/1> a schema:DefinedTerm ;
```

Comment:
We already have skos:Concept defined separately but don't use them. DefinedTerm for concepts tagged in the text and skos:Concept for the explicitly defined hierarchies *can* coexist but it could be confusing. Thoughts?

The proper approach might be to define custom subclasses of skos:Concept but the platform does not support objects with more than one rdf type so we can't use class inheritance due to technological limitations.

Reply:
in schema this is supposed to be done via schema:additionalType and you should have separate URIs. 
<Location/1> a schema:DefinedTerm ; schema:additionalType <locations/Location/1> .
<locations/Location/1> a skosConcept .... 
What a mess eh